docs/concepts/api-extension/custom-resources.md

```diff
@@ -42,12 +43,162 @@ The [Operator](https://coreos.com/blog/introducing-operators.html) pattern is on
 combination. It allows developers to encode domain knowledge for specific applications into an
 extension of the Kubernetes API.
 
+### Should I Add a Custom Resource to my Kubernetes Cluster?
+When creating a new API, first consider if you should add your API to Kubernetes cluster APIs, or instead let your API stand alone.
+
+
+<table>
```

Comment:
Something went wrong in the rendering: https://deploy-preview-6238--kubernetes-io-master-staging.netlify.com/docs/concepts/api-extension/custom-resources/#should-i-add-a-custom-resource-to-my-kubernetes-cluster